packages/shared/pkg/storage/header/mapping.go

```diff
@@ -39,9 +38,9 @@ func CreateMapping(
 
 		if blockLength > 0 {
 			m := &BuildMap{
-				Offset:             uint64(int64(startBlock) * int64(metadata.BlockSize)),
+				Offset:             uint64(int64(startBlock) * int64(blockSize)),
 				BuildId:            *buildId,
```

Comment:
(Was there before) Unnecessary conversion
```suggestion
				Offset:             uint64(startBlock) * blockSize,
```